StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -104,7 +104,9 @@ final class UpdatePaymentMethodViewController: UIViewController {
     }()
 
     private lazy var setAsDefaultCheckbox: CheckboxElement? = {
-        guard viewModel.canSetAsDefaultPM && PaymentSheet.supportedDefaultPaymentMethods.contains(where: {
+        guard viewModel.canSetAsDefaultPM,
```

Comment:
It seems like canSetAsDefaultPM should check isDefault so it's more of the single source of truth, otherwise it reads funny that we need to check something else if it's able to be same with the supportedDefaultPaymentMethods check